client/src/containers/Courses/Courses.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import { getCourses } from "../../helpers/api";
+class Courses extends Component {
+  state = {
+    courses: []
+  };
+  componentDidMount() {
+    getCourses().then(data => {
+      this.setState({ courses: JSON.stringify(data) });
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Courses table</p>
+        {/* <table>
+          <thead>
+            <tr>
+              <th>organisationName</th>
+              <th>courseTitle</th>
+              <th>lessonName</th>
+              <th>location</th>
+            </tr>
+          </thead>
+          <tbody>
+            {console.log(this.state.courses)}
+            {this.state.courses.map(result => (
+              <tr key="1">
+                <td>{result.organisationName}</td>
+                <td>{result.courseTitle}</td>
+                <td>{result.lessonName}</td>
+                <td>{result.location}</td>
+              </tr>
+            ))}
+          </tbody>
+        </table> */}
```

Comment:
Do we need the above comment or can we remove it?

Reply:
Yes, I need them because there is an error for getting the data as array.